pkg/bindings/util/util.go

```diff
@@ -11,14 +12,23 @@ import (
 )
 
 func IsSimpleType(f reflect.Value) bool {
+	if _, ok := f.Interface().(fmt.Stringer); ok {
+		return true
+	}
+
 	switch f.Kind() {
 	case reflect.Bool, reflect.Int, reflect.Int64, reflect.Uint, reflect.Uint64, reflect.String:
 		return true
 	}
+
 	return false
 }
 
 func SimpleTypeToParam(f reflect.Value) string {
```

Comment:
Just occurred to me that the function could now look like:
```go
func SimpleTypeToParam(f reflect.Value) string {
	if IsSimpleType(f) {
		return fmt.Sprintf("%v", f.Interface())
	}
	panic("the input parameter is not a simple type")
}
```
But no need to change it now.